programs/svm-spoke/src/instructions/slow_fill.rs

```diff
@@ -35,15 +33,15 @@ pub struct SlowFillV3Relay<'info> {
         seeds = [b"fills", relay_hash.as_ref()],
         bump,
         // Make sure caller provided relay_hash used in PDA seeds is valid.
-        constraint = is_relay_hash_valid(&relay_hash, &relay_data, &state) @ CustomError::InvalidRelayHash
+        constraint = is_relay_hash_valid(&relay_hash, &relay_data, &state) @ SvmError::InvalidRelayHash
     )]
     pub fill_status: Account<'info, FillStatusAccount>,
     pub system_program: Program<'info, System>,
 }
 
 pub fn request_v3_slow_fill(
     ctx: Context<SlowFillV3Relay>,
-    relay_hash: [u8; 32], // include in props, while not using it, to enable us to access it from the #Instruction Attribute within the accounts. This enables us to pass in the relay_hash PDA.
+    _: [u8; 32], // include in props, while not using it, to enable us to access it from the #Instruction Attribute within the accounts. This enables us to pass in the relay_hash PDA.
```

Comment:
same comment as in others, rename it to `_relay_hash` in the outer call and not pass it to this inner function